webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js

```diff
@@ -1,89 +1,105 @@
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { selectDoesIntervalExist } from 'foremanReact/redux/middlewares/IntervalMiddleware/IntervalSelectors';
 import { selectForemanTasks } from '../../ForemanTasksSelectors';
+import { FOREMAN_TASK_DETAILS } from './TaskDetailsConstants';
 
 export const selectTaskDetails = state =>
   selectForemanTasks(state).taskDetails || {};
 
-export const selectStartAt = state => selectTaskDetails(state).start_at || null;
+export const selectTaskDetailsResponse = state =>
+  selectAPIResponse(state, FOREMAN_TASK_DETAILS);
+
+export const selectStartAt = state =>
+  selectTaskDetailsResponse(state).start_at || null;
 
 export const selectStartBefore = state =>
-  selectTaskDetails(state).start_before || null;
+  selectTaskDetailsResponse(state).start_before || null;
 
 export const selectStartedAt = state =>
-  selectTaskDetails(state).started_at || null;
+  selectTaskDetailsResponse(state).started_at || null;
 
-export const selectEndedAt = state => selectTaskDetails(state).ended_at || null;
+export const selectEndedAt = state =>
+  selectTaskDetailsResponse(state).ended_at || null;
```

Comment:
want to use `optional chaining` here? it will return `undefined` if the response doesn't exist

Reply:
changed, had to add `/* eslint-disable camelcase */` to the file to use it